products/secretmanager/terraform.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Secret: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "secret_config_basic"
+        primary_resource_id: "secret-basic"
+        primary_resource_name: "fmt.Sprintf(\"test-secret-basic%s\", context[\"random_suffix\"])"
+        min_version: beta
+        vars:
+          secret_id: "tf-test-secret-"
+    properties:
+      replication.automatic: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_flatten: templates/terraform/custom_flatten/object_to_bool.go.erb
+        custom_expand: templates/terraform/custom_expand/bool_to_object.go.erb
+
+  SecretVersion: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
How will `import` work for this resource if the important part (payload.data) is `ignore_read`?

Reply:
I'm not sure.  I just wrote the test, and it doesn't work.  Hopefully, if I can figure out how to deal with the `:access`, `:enable` and `:disable` on this resource, it will solve this as well.

Reply:
Possibly a custom flatten on `payload.data` that calls the `:access` endpoint?

We have `delete_url` that you can specify on a resource, but I'm not sure how the `:enable` and `:disable` actually work for this API